src/labels/CustomLabels.labels

```diff
@@ -1318,6 +1318,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Processing</shortDescription>
         <value>Processing</value>
     </labels>
+    <labels>
+        <fullName>RD2_DataMigrationInvalidInstallmentPeriod</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Installment Period cannot be converted to enhanced Recurring Donation format</shortDescription>
+        <value>Recurring Donation Installment Period "{0}" cannot be converted to enhanced Recurring Donation format.</value>
```

Comment:
@sam-knox , can you please advise on the message displayed when Installment Period cannot be converted?

Reply:
@RadSFDeveloper - What would the advice be to a user in this scenario? Should they re-create manually somehow, or are we saying that certain custom installment periods simply won't work in RD2?

Reply:
The custom installment needs to conform to "Days", "Weeks", "Months" and "Years". I assume users would need to update their RDs  to those installments. @force2b , thoughts?

Reply:
I'm not sure what you mean for this? Custom Installments have a Name field that the Admin adds to the Picklist. RD2 post installment instructions will have to include a note about asking them to remove those picklist values from the field, but the User shouldn't need to change the RD since migration will apply the converted schedule type to the record.
https://powerofus.force.com/s/article/NPSP-Installmt-Periods-Recur-Dons

Reply:
@force2b - So does this mean we don't need an error message for this scenario?

Reply:
An error is still possible if there is a Picklist value in the InstallmentPeriod field that does not map to something we can understand. That would be Weekly, Monthly, Yearly, Quarterly, 1st and 15th, and any custom names that are defined in the CustomInstallmentPeriod settings object. For example, if they create a picklist value called "Eventually" and there is no record in CustomInstallmentPeriod with a name of "Eventually" that would generate an error.

Reply:
@sam-knox , the change to the text message has been applied.